aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Commons.java

```diff
@@ -126,16 +126,16 @@ public static <M, C> ProgressEvent<M, C> reportResourceDrift(
             final M inputModel,
             final ProgressEvent<M, C> progress,
             final ResourceTypeSchema schema,
-            final RequestLogger logger
-    ) {
+            final RequestLogger requestLogger,
+            final String handlerAction) {
         try {
             final DriftDetector driftDetector = new DriftDetector(schema);
             final Map<String, Mutation> mutations = driftDetector.detectDrift(inputModel, progress.getResourceModel());
             if (!mutations.isEmpty()) {
-                logger.log("Resource drift detected", new DriftDetectorReport(mutations));
+                requestLogger.log(String.format("Resource drift detected! Action: %s", handlerAction), new DriftDetectorReport(mutations));
```

Comment:
nit: Could use structured logging for the ease of search:
`requestLogger.log("Resource drift detected", ImmutableMap.of("Operation", handlerAction, "Report", new DriftDetectorReport(mutations)));